### PR TITLE
Update summary

### PR DESCRIPTION
**Pull request by mayou36**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mayou36** [comment]
@conda-forge-admin, please rerender

**Comment by chrisburr** [comment]
Did you mean to close this?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,14 +45,14 @@ test:
     - zfit.minimizers
     - zfit.models
     - zfit.util
-    - zfit.ztf
+    - zfit.z
```
Comment (chrisburr) [inline recipe/meta.yaml]:
```suggestion
     - zfit.ztf
```

Comment (mayou36) [reply recipe/meta.yaml]: Yes, I meant to close it, I will apply the changes in the new version.

**Comment by mayou36** [comment]
Sorry for the noise :)

**Pull request closed without merging**